code/obj/machinery/computer/QM_supply.dm

```diff
@@ -1229,13 +1252,7 @@ var/global/datum/cdc_contact_controller/QM_CDC = new()
 					</li>
 					"}
 
-			bottomText += {"
-				</ul>
-				<br>
-				<br><em>To sell goods to this trader, print a barcode for <strong>[T.name]</strong> on the barcode computer, attach it to a crate containing the goods, and send the crate out the 'sell' mass driver.
-				<br>
-				<br>Load no more than 50 items into a crate at once, or the trader's cargo computer may not be able to keep up!</em>
```

Comment:
why rempove?

Reply:
the 50-item limit doesn't exist; re-added the note on how to sell